Automattic/wp-calypso
# Remove the "Set primary domain" dedicated page as it's not used anymore

#### Changes proposed in this Pull Request

* We're not using the dedicated "Set primary domain" page anymore so no need for it to exist in our code

#### Testing instructions

* check that the domain status pages are still working and that you can still change the primary domain
